packages/visual-editor/src/components/pageSections/NearbyLocations.tsx

```
@@ -470,12 +473,50 @@ const NearbyLocationsComponent = ({
       }[styles.heading.align]
     : "justify-start";
 
-  // do not render the component if there's no data or it's not enabled
+  // Show empty state in editor mode when there are no results
   if (
     !enableNearbyLocations ||
     (!nearbyLocationsData?.response?.docs?.length &&
       nearbyLocationsStatus != "pending")
   ) {
+    if (puck.isEditing) {
+      const templateMetadata = React.useContext(TemplateMetadataContext);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)